### PR TITLE
Modify and add tests for add, delete, and find commands

### PR DESCRIPTION
**Pull request by elhy1999**
Most changes proposed are minor changes such as names (e.g. person --> transaction, addressbook --> unicash).

Here are some other changes proposed:
FindCommandTest now is able to test if the command can filter >1 transactions that contain a keyword. To do this, I modified `CommandTestUtil#VALID_TRANSACTION_NAME_INTERN` from "Internship" to "Internship work". This was done so that now, two transactions contain the keyword "work". The `NUS` transaction contains the word "Work" (uppercase), while the `INTERN` transaction contains the keyword "work" (lowercase). Hence, this test also checks if the find command is case-insensitive, which is the desired behavior.

### CODE REVIEW

File: src/test/java/unicash/model/ModelManagerTest.java
Comment (sp4ce-cowboy) [inline src/test/java/unicash/model/ModelManagerTest.java:138]: I think this comment can be removed!

File: src/test/java/unicash/testutil/TypicalTransactions.java
Comment (sp4ce-cowboy) [inline src/test/java/unicash/testutil/TypicalTransactions.java:79]: Nice work catching this minor detail! Can we further change it to just uniCash or emptyUniCash to better align with the variable naming conventions? I doubt it would be an issue though, just trying to be extra careful with the coding convention requirements

**Review by sp4ce-cowboy — APPROVED** [review]
LGTM

File: src/test/java/unicash/model/ModelManagerTest.java
Comment (elhy1999) [reply src/test/java/unicash/model/ModelManagerTest.java:138]: I think that was added by @woojiahao. Can this be removed, Jia Hao? If so, I will remove it before merging the PR.

File: src/test/java/unicash/testutil/TypicalTransactions.java
Comment (elhy1999) [reply src/test/java/unicash/testutil/TypicalTransactions.java:79]: Good suggestion! I've modified it to `typicalUnicash` instead.

File: src/test/java/unicash/model/ModelManagerTest.java
Comment (elhy1999) [reply src/test/java/unicash/model/ModelManagerTest.java]: ^ Jia Hao has confirmed that it can be removed. Merging this PR will remove that line of comment

File: src/test/java/unicash/logic/commands/AddTransactionCommandTest.java
Comment (woojiahao) [inline src/test/java/unicash/logic/commands/AddTransactionCommandTest.java:62]: Could we use the pre-built `NUS` and `INTERN` global instances? So there's no accidental changes that are made that aren't being captured

**Review by woojiahao — CHANGES_REQUESTED** [review]
Mostly LGTM! But just wondering if we should use `NUS` or `INTERN` instead of recreating again

File: src/test/java/unicash/logic/commands/AddTransactionCommandTest.java
Comment (elhy1999) [reply src/test/java/unicash/logic/commands/AddTransactionCommandTest.java]: Solved! Thanks for pointing this out

**Review by woojiahao — APPROVED** [review]
Awesome, LGTM, thanks!!

**Pull request merged**
